variables.tf

```
@@ -101,6 +101,24 @@ variable "network_rule_set_trusted_services_allowed" {
   nullable    = false
 }
 
+variable "queue" {
+  description = "Manages a ServiceBus Queue."
```

Name should be plural since value type is a collection:

```suggestion
variable "queues" {
  description = "A map of queues to create for this Service Bus namespace."
```

Also updated description to be more in line with how we usually write descriptions.

Default object property values look good to me 😊

---

References to this variable must be updated to reference this new name as well.